app/assets/stylesheets/hybox.scss

```diff
@@ -0,0 +1,16 @@
+$brand-primary: #629DD1 !default;
+$jumbotron-heading-font-size: 2.5em;
+$jumbotron-heading-color: #fff;
+$jumbotron-color: #d8d8d8;
+
+.jumbotron {
+  background-image: url("http://hydrainabox.projecthydra.org/images/dark_tint.png"), url("http://hydrainabox.projecthydra.org/images/fractal.jpg");
```

Comment:
I would if I was convinced this was a long term solution. My intent was to not bloat the repo with binary assets that will likely be replaced soon.